### DISCUSSION TITLE
No motion controllers and camera is above the menu when starting game with SubmersedVR

### DISCUSSION THREAD

**Discussion by ThrowawayTheSecond** [Okabintaro/SubmersedVR#50] [Q&A] [answered]
When I run the game in steamVR and stream to oculus quest 2 via Airlink with this mod, I have no motion controllers and none of the buttons on the controllers are working, so I am unable to select anything from the menu. The vr camera is also above the menu, recentering places the camera in the correct position.
![screenshot](https://github.com/Okabintaro/SubmersedVR/assets/138209527/37153119-156c-4ef3-a02c-1056aada7d19)

**Comment by Okabintaro** [comment]
Please make sure to follow the README guide properly. You most likely forgot to set the launch commands on steam.
If not please upload a logfile.

**Comment by ThrowawayTheSecond** [comment]
I did write -vrmode openvr into the subnautica launch options, where do I find a logfile?

**Comment by Okabintaro** [comment]
It is in `Subnautica\BepInEx\LogOutput.txt`. You might also want to join the [Flat2VR](http://flat2vr.com) for better support.
The issues here are used for feature requests and bug reports. Thanks!
Alternatively you could also use the [discussions](https://github.com/Okabintaro/SubmersedVR/discussions).

**Comment by ThrowawayTheSecond** [comment]
[LogOutput.log](https://github.com/Okabintaro/SubmersedVR/files/11919972/LogOutput.log)
This is the file, sorry if this is not the right place for this.

**Reply by Okabintaro** [reply, marked as answer]
It seems you haven't properly installed the mod. You need to copy the whole folder into the subnautica folder like this: ![](https://cdn.discordapp.com/attachments/754030896130031736/1115433004165251103/image.png)

**Reply by ThrowawayTheSecond** [reply]
I reinstalled the mod and it works. Thanks!

**Comment by Ventras26** [comment]
i am not having the Spawn issue but i also have No hands and no controlls 

BepInEx had an Update 2 weeks ago 
heres my log 
Please help me i got so exited when i saw this mod 

[LogOutput.log](https://github.com/Okabintaro/SubmersedVR/files/11925960/LogOutput.log)

**Reply by Okabintaro** [reply]
Apperently you didn't even start the game in VR:

```
[Warning:SubmersedVR] Game was not started in VR. Don't load and apply SubmersedVR patches.
[Error  :SubmersedVR] SubmersedVR only supports SteamVR! Make sure to add `-vrmode openvr` to your games advanded launch options when using an oculus headset.
```

**Comment by mcses7** [comment]
> Apperently you didn't even start the game in VR:
> 
> ```
> [Warning:SubmersedVR] Game was not started in VR. Don't load and apply SubmersedVR patches.
> [Error  :SubmersedVR] SubmersedVR only supports SteamVR! Make sure to add `-vrmode openvr` to your games advanded launch options when using an oculus headset.
> ```
i started mine in vr and followed the guide on the readme but it still wont work
